scopesim/utils.py

```diff
@@ -414,38 +418,82 @@ def set_logger_level(which="console", level="ERROR"):
             hdlr.setLevel(level)
 
 
-def bug_report():
-    """Get versions of dependencies for inclusion in bug report."""
-    try:
-        from importlib import import_module
-    except ImportError:
-        import_module = __import__
+def _get_required_packages():
+    reqs = metadata.requires(__package__)
+    for req in reqs:
+        # Only include non-extra packages
+        if "extra" in req:
+            continue
+
+        name = req.split(">", maxsplit=1)[0]#.replace("-", "_")
```

Comment:
```suggestion
        name = req.split(">", maxsplit=1)[0].strip()
```
otherwise name can be `'astropy '`, with a trailing space